ci/cudf_pandas_scripts/pandas-tests/run.sh

```diff
@@ -37,6 +37,7 @@ bash python/cudf/cudf/pandas/scripts/run-pandas-tests.sh \
   -m "not slow" \
```

Comment:
*suggestion*:

I found that `--tb=no` is much faster (and produces less output) than the other options.

Reply:
Done